hca/dss/__init__.py

```diff
@@ -93,34 +93,52 @@ def _create_version(self):
         return datetime.utcnow().strftime("%Y-%m-%dT%H%M%S.%fZ")
 
     def upload(self, src_dir, replica, staging_bucket, timeout_seconds=1200, no_progress=False,
-               bundle_uuid=None):
+               bundle_uuid=None, file_uuids_config=None):
```

Comment:
Consider allowing the caller to alternatively specify a `dict`. When calling this directly from a script (not through the CLI) it would be a pain if I had to allocate a temporary file.